tests/e2e/tests/test_storage_efs.py

```diff
@@ -148,13 +149,13 @@ def test_pvc_with_volume(
         subprocess.run(f"kubectl delete pod -n {DEFAULT_USER_NAMESPACE} {write_pod_name}".split())
         subprocess.run(f"kubectl delete pod -n {DEFAULT_USER_NAMESPACE} {read_pod_name}".split())
 
-
+"""
```

Comment:
why is this commented out?

Reply:
let me uncomment it out again. Basically Meghna combine the dynamic test and static test together and it requires two clusters, so I commented out one test and run it one by one. We should decouple this test in the future.